src/engine/blocks.js

```diff
@@ -399,7 +399,7 @@ Blocks.prototype.blockToXML = function (blockId) {
     // Encode properties of this block.
     var tagName = (block.shadow) ? 'shadow' : 'block';
     var xy = (block.topLevel) ?
-        ' x="' + block.x +'"' + ' y="' + block.y +'"' :
+        ' x="' + block.x +'" y="' + block.y +'"' :
```

Comment:
This is a curious fix. It makes sense, but I'm surprised we aren't enforcing spaces around operators (e.g. `'foo' + 'bar'` vs `'foo'+ 'bar'`) like we are elsewhere (e.g. `function ()`)